docs/faq.md

```diff
@@ -0,0 +1,87 @@
+
+    1. Uninstall existing mmcv in the environment using `pip uninstall mmcv`.
+    2. Install mmcv-full following the [installation instruction](https://mmcv.readthedocs.io/en/latest/#installation).
+
+
+## Data
+
+
+- **What if my custom dataset does not have bounding box label?**
+
+    We can estimate the bounding box of a person as the minimal box that tightly bounds all the keypoints.
+
+- **What if my custom dataset does not have segmentation label?**
+
+    Just set the `area` of the person as the area of the bounding boxes. During evaluation, please set `use_area=False` as in this [example](https://github.com/open-mmlab/mmpose/blob/a82dd486853a8a471522ac06b8b9356db61f8547/mmpose/datasets/datasets/top_down/topdown_aic_dataset.py#L113).
+
+- **What is `COCO_val2017_detections_AP_H_56_person.json`? Can I train pose models without it?**
+
+    "COCO_val2017_detections_AP_H_56_person.json" contains the "detected" human bounding boxes for COCO validation set, which are generated by FasterRCNN.
+    One can choose to use gt bounding boxes to evaluate models, by setting `use_gt_bbox=True` and `bbox_file=''`. Or one can use detected boxes to evaluate
+    the generalizability of models, by setting `use_gt_bbox=False` and `bbox_file='COCO_val2017_detections_AP_H_56_person.json'`.
+
+- **How to evaluate on MPII test dataset?**
+    Since we do not have the ground-truth for test dataset, we cannot evalute it 'locally'.
+    If you would like to evaluate the performance on test set, you have to upload the pred.mat (which is generated during testing) to the official server via email, according to [the MPII guideline](http://human-pose.mpi-inf.mpg.de/#evaluation).
+
+## Training
+
+- **RuntimeError: Address already in use**
+
+    Set the environment variables `MASTER_PORT=XXX`. For example,
+    `MASTER_PORT=29517 GPUS=16 GPUS_PER_NODE=8 CPUS_PER_TASK=2 ./tools/slurm_train.sh Test res50 configs/top_down/resnet/coco/res50_coco_256x192.py work_dirs/res50_coco_256x192`
+
+- **How to use trained models for backbone pre-training ?**
+
+    Refer to [Use Pre-Trained Model](https://github.com/open-mmlab/mmpose/blob/master/docs/tutorials/1_finetune.md#use-pre-trained-model),
+    in order to use the pre-trained model for the whole network (backbone + head), the new config adds the link of pre-trained models in the `load_from`.
+
+    And to use backbone for pre-training, you can change `pretrained` value in the backbone dict of config files to the checkpoint path / url.
+    When training, the unexpected keys will be ignored.
+
+- **How to visualize the training accuracy/loss curves in real-time ?**
+
+    Use `TensorboardLoggerHook` in `log_config` like
+    ```python
+    log_config=dict(interval=20, hooks=[dict(type='TensorboardLoggerHook')])
+    ```
+    You can refer to [tutorials/6_customize_runtime.md](tutorials/6_customize_runtime.md#log-config) and the example [config](/configs/top_down/resnet/coco/res50_coco_256x192.py#L26).
+
+
```

Comment:
Add a section `Evaluation` and give instructions on how to evaluate the performance on MPII test set? https://github.com/open-mmlab/mmpose/issues/300#issuecomment-733428238